src/main/resources/templates/groups.html

```diff
@@ -17,7 +17,7 @@
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
                 integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
                 crossorigin="anonymous"></script>
-        <script language="javascript" type="text/javascript" src="../static/script.js" th:src="@{/script.js}"></script>
+        <script src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```

Comment:
```suggestion
        <script type="text/javascript" src="../static/js/script.js" th:src="@{/js/script.js}"></script>
```
Bare minimum. Not all browsers autodetect JS.

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script the type and language attributes should not be used

Reply:
Stupid HTML5, I'm too old for this.